### PR TITLE
Copy chairman app to new cabal project

### PR DESCRIPTION
**Pull request by newhoggy**
This allows for the changes to be made to the new chairman app without breaking the old chairman tests.

### CODE REVIEW

File: cardano-node-chairman/app/Main.hs [hunk 1]
```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE NamedFieldPuns #-}
```
Comment (erikd) [inline cardano-node-chairman/app/Main.hs:1]: Should rename that to maybe `chariman.hs`. The leading lower case shows that it its a Haskell main and not a module.

**Review by erikd — APPROVED** [review]
Just think we need a rename but once that is done all good.

File: cardano-node-chairman/app/Main.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node-chairman/app/Main.hs:1]: Should I use `cardano-node-chairman.hs` for now, since that's the name of the exe?

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**